src/modules/fancyzones/lib/WindowMoveHandler.cpp

```diff
@@ -316,7 +318,13 @@ void WindowMoveHandlerPrivate::MoveSizeEnd(HWND window, POINT const& ptScreen, c
     {
         auto zoneWindow = std::move(m_zoneWindowMoveSize);
         ResetWindowTransparency();
-        zoneWindow->MoveSizeEnd(m_windowMoveSize, ptScreen);
+
+        auto windowInfo = FancyZonesUtils::GetFancyZonesWindowInfo(window);
+        if (windowInfo.standardWindow == m_moveSizeStartWindowInfo.standardWindow &&
+            windowInfo.noVisibleOwner == m_moveSizeStartWindowInfo.noVisibleOwner)

```

Comment:
Let's do:
```
if (windowInfo.standardWindow == false && windowInfo.noVisibleOwner == false &&   m_moveSizeStartWindowInfo.standardWindow == true && m_moveSizeStartWindowInfo.noVisibleOwner == true)
{
    // Abort the zoning, this is a Chromium based tab that is merged back with an existing window
}
```
So the check is 100% specific for the Chrome tab case and doesn't cover the other way around (that we don't even know if it's actually possible, but if it is, it's not currently broken)

Reply:
other way around isn't possible, as if standardWindow was false in moveSizeStart, we wouldn't get here at all

Reply:
Still, I prefer the code covers the specific case, this is for long term code maintainability, it should only check what we know it should check and not cover other cases.